pandas/core/computation/eval.py

```diff
@@ -206,19 +205,32 @@ def eval(expr, parser='pandas', engine=None, truediv=True,
         The number of prior stack frames to traverse and add to the current
         scope. Most users will **not** need to change this parameter.
     target : a target object for assignment, optional, default is None
```

Comment:
this doc-string is not numpy doc (IOW the expl should be on the next line), specify *what* the target object can be (DataFrame?)